apps/ui/src/views/account/AccountProfileView.vue

```diff
@@ -0,0 +1,216 @@
+        <p class="text-center" style="color: white" >Loading Profile...</p>
+      </v-progress-circular>
+    </div>
+    <div v-else-if="!userData.person">
+      <v-form ref="profileCreation">
+        <p>You currently do not have a profile, you can create it by entering your name below.</p>
+        <v-text-field :rules="[nameRule]" label="Name" v-model="profileName" />
+        <v-btn @click="createProfile" class="create-profile" color="green">Create Profile</v-btn>
+      </v-form>
+    </div>
+    <div v-else>
+      <h1 >Profile Information</h1>
+      <div class="details">
+        <p class="label">Name: <span class="info">{{ personData.name }}</span></p>
+        <v-btn @click="editableValues.name.dialog = true;" variant="text" icon="fa-solid fa-pen-to-square" />
+        <v-dialog :max-width="500" v-model="editableValues.name.dialog">
+          <template #default>
+            <v-card title="Edit Name" :max-width="500">
+              <template #text>
+                <v-text-field label="Enter your name" v-model="editableValues.name.value" />
+              </template>
+              <template #actions>
+                <v-spacer />
+                <v-btn :disabled="personData.name === editableValues.name.value" @click="() => {
+                  personData.name = editableValues.name.value;
+                  saveChanges('Successfully edited name');
+                  editableValues.name.dialog = false;
+                }">SAVE</v-btn>
+              </template>
+            </v-card>
+          </template>
+        </v-dialog>
+      </div>
+      <div class="details">
+        <p class="label">Pronouns:
+          <span v-if="userData.person.pronouns?.trim().length" class="info">{{ personData.pronouns }}</span>
+          <span v-else><em>No pronouns set</em></span>
+        </p>
+        <v-btn @click="editableValues.pronouns.dialog = true;" variant="text" icon="fa-solid fa-pen-to-square" />
+        <v-dialog :max-width="500" v-model="editableValues.pronouns.dialog">
+          <template #default>
+            <v-card title="Edit Pronouns" :max-width="500">
+              <template #text>
+                <v-combobox label="Enter your pronouns" :items="pronouns" v-model="editableValues.pronouns.value" />
+              </template>
+              <template #actions>
+                <v-spacer />
+                <v-btn :disabled="personData.pronouns === editableValues.pronouns.value" @click="() => {
+                  personData.pronouns = editableValues.pronouns.value;
+                  saveChanges('Successfully edited pronouns');
+                  editableValues.pronouns.dialog = false;
+                }">SAVE</v-btn>
+              </template>
+            </v-card>
+          </template>
+        </v-dialog>
+      </div>
+      <div class="details">
+        <p class="label">Graduation Date:
+          <span v-if="personData.graduation" class="info">{{ moment(personData.graduation).format("LL") }}</span>
```

Comment:
Graduation date is being displayed as 12-31-1969 for me immediately after creating my profile, despite being NULL in the db

Reply:
Fixed